chainer/iterators/multiprocess_iterator.py

```diff
@@ -62,13 +62,17 @@ def __init__(self, dataset, batch_size, repeat=True, shuffle=True,
 
         self._finalized = None
 
+        self._previous_epoch_detail = -1.
```

Comment:
Sorry for bringing up the problem again, but I am wondering why we choose `None` instead of `-1` as the initial value of `previous_epoch_detail`. I thought the other alternative is to define `-1` as an invalid value of `previous_epoch_detail` (e.g. in the document of `Iterator`) and not to convert the value internally. I am sorry if I miss some previous discussion and make a non-sense question.